resources/qml/Sidebar.qml

```diff
@@ -191,23 +277,152 @@ Rectangle
         }
     }
 
-    Rectangle {
+    // Item that shows the print monitor properties
+    Column
```

Comment:
Can we please move this to its own file? Just dumping it in here is rather ugly. Additionally, we would be able to use loader to avoid constantly having this in memory.